invenio_communities/views/api.py

```diff
@@ -177,9 +181,21 @@ def get(self, community_id):
         community = Community.get(community_id)
         if not community:
             abort(404)
-        return self.make_response(
-                community, links_item_factory=default_links_item_factory)
 
+        etag = hashlib.sha1(
```

Comment:
@jirikuncar With name ``version_id`` it could be a bit confusing due to the ``etag`` is identifying uniquely a community && a specific status (I understand that the version_id could be the same in several communties). On the other hand, having a ``version_id`` field, the method is more intuitive when you are generating the ``etag`` (community.id + community.version_id). In my opinion, what ``Community`` model needs is a ``hash`` function.